tests/pytype_test.py

```diff
@@ -191,37 +188,35 @@ def get_missing_modules(files_to_test: Sequence[str]) -> Iterable[str]:
     Similarly, pytype cannot parse files on its exclude list, so we also treat
     those as missing.
     """
-    stub_distributions = set()
+    stub_distributions = set[str]()
     for fi in files_to_test:
-        parts = fi.split(os.sep)
+        parts = fi.parts
         try:
             idx = parts.index("stubs")
         except ValueError:
             continue
         stub_distributions.add(parts[idx + 1])
 
-    missing_modules = set()
-    for distribution in stub_distributions:
-        for external_req in read_dependencies(distribution).external_pkgs:
-            associated_packages = _get_pkgs_associated_with_requirement(external_req.name)
-            missing_modules.update(associated_packages)
-
-    test_dir = os.path.dirname(__file__)
-    exclude_list = os.path.join(test_dir, "pytype_exclude_list.txt")
-    with open(exclude_list) as f:
-        excluded_files = f.readlines()
-        for fi in excluded_files:
-            if not fi.startswith("stubs/"):
+    missing_modules = {
+        associated_package
+        for distribution in stub_distributions
+        for external_req in read_dependencies(distribution).external_pkgs
+        for associated_package in _get_pkgs_associated_with_requirement(external_req.name)
+    }
+
+    with EXCLUDE_LIST.open() as f:
+        for line in f:
+            if not line.startswith("stubs/"):
                 # Skips comments, empty lines, and stdlib files, which are in
                 # the exclude list because pytype has its own version.
                 continue
-            unused_stubs_prefix, unused_pkg, mod_path = fi.split("/", 2)  # pyright: ignore[reportUnusedVariable]
-            missing_modules.add(os.path.splitext(mod_path)[0])
+            _ts_subdir, _distribution, module_path = line.split("/", 2)
+            missing_modules.add(module_path.rsplit(".", 1)[0])
```

Comment:
This could also be:
```suggestion
            missing_modules.add(module_path.removesuffix(".pyi"))
```

Reply:
I agree, to me as well.

Reply:
Oh, haha, I know why this breaks. The suffix is `.pyi\n` 😅